tests/cases/fourslash/jsDocVariadicType.ts

```diff
@@ -0,0 +1,16 @@
+///<reference path="fourslash.ts" />
+
+// @allowJs: true
+// @checkJs: true
+
+// @Filename: dummy.js
+//// /**
+////  * @type {function(boolean, string, ...*):void}
+////  */
+//// const foo = function (a, b, ...r) { };

```

Comment:
Does this error repro from the command line `tsc`? It should, and if so, I'd much prefer a test in `tests/conformance/jsdoc/`. Without the fix there should be a new .errors.txt baselines, with the fix there shouldn't be.

Reply:
Thanks for pointing that out, I pushed a commit moving the test to conformance